application/common/models/User.php

```diff
@@ -168,7 +168,18 @@ public function updateProfileIfNeeded($firstName, $lastName, $username, $email)
      */
     public function getMaskedMethods()
     {
-        $methods = [];
+        /*
+         * Include primary email address
```

Comment:
Do we also need to manually include the supervisor and spouse options (when applicable) here?